docs/topics/mta-web-skipping-assessment-of-apps.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-skipping-assessment-of-apps_{context}"]
+= Skipping assessment of applications
+
+By default, the Assessment procedure is mandatory, and you cannot proceed to the next step, Reviewing, without completing Assessment first. However, you might want to skip Assessment and use your own questionnaire instead. In that case, use the option described below.
```

Comment:
**MTA <6.1** - By default, the Assessment procedure is a prerequisite for application review.
**MTA 6.1**   - Starting with MTA 6.1 , by default, the Assessment procedure is not mandatory  . The reason being the 
                       *Allow reviewing applications without running an assessment first* option is enabled by default upon 
                        installation of MTA.

Reply:
@nachandr 
I've rephrased it.

Reply:
Thank you !